stdlib/3.7/contextvars.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Callable, ClassVar, Generic, Mapping, TypeVar, Union
+
+_T = TypeVar('_T')
+
+class ContextVar(Generic[_T]):
+    def __init__(self, name: str, *, default: _T) -> None: ...
```

Comment:
`default` is not a required parameter.  It's not `typing.Optional` either, as it is either of type `_T` or not passed at all (`None` is not a valid value).  I'm not sure how to express this in terms of `typing`.

A couple of examples:

```python
# valid:
int_var: ContextVar[int] = ContextVar(default=1)
int_var: ContextVar[int] = ContextVar()

# invalid:
int_var: ContextVar[int] = ContextVar(default=None)
```

and

```python
# valid:
int_var: ContextVar[Optional[int]] = ContextVar(default=1)
int_var: ContextVar[Optional[int]] = ContextVar(default=None)
int_var: ContextVar[Optional[int]] = ContextVar()
```

Here's a pure Python implementaion of PEP 567 / `ContextVar`: https://github.com/MagicStack/contextvars/blob/master/contextvars/__init__.py#L111

Reply:
We can write `*, default: _T = ...`. Sorry for missing this.

Reply:
Jelle: If `= ...` doesn't work, you can use an overload (once without, once with `default`).

Reply:
NP!